docs/content/en/docs/packages/packagecontroller.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Manging the package controller"
+linkTitle: "Package controller"
+weight: 4
+aliases:
+    /docs/tasks/packages/packagecontroller/
+---
+
+### Install package controller after installation
+
+{{% alert title="Important" color="warning" %}}
+**Recommendation**: Ideally, you should never delete package controller after installation. When the package bundle controller resource is deleted and then gets recreated, the active bundle recorded in the package bundle controller resource will point to the latest bundle, but you can always go back to the previous bundle version. See more information about managing package bundles [here]({{< relref "./packagebundles" >}})
+{{% /alert %}}
+
+
+If you created a cluster without the package controller or if the package controller was not properly configured, you may need to do some things to enable it.
+
+Make sure you are authenticated with the AWS CLI. Use the credentials you set up for packages. These credentials should have [limited capabilities]({{< relref "./prereq#setup-authentication-to-use-curated-packages" >}}):
+
+```bash
+export AWS_ACCESS_KEY_ID="your*access*id"
+export AWS_SECRET_ACCESS_KEY="your*secret*key"
+export EKSA_AWS_ACCESS_KEY_ID="your*access*id"
+export EKSA_AWS_SECRET_ACCESS_KEY="your*secret*key"
+```
+
+Verify your credentials are working:
+```shell
+aws sts get-caller-identity
+```
+Login to docker
+```bash
+aws ecr get-login-password |docker login --username AWS --password-stdin 783794618700.dkr.ecr.us-west-2.amazonaws.com
```

Comment:
what is 783794618700 account?

Reply:
It looks like you are using aws credentials to have docker login to ECR to gain access to the registry that holds the EKS Anywhere packages artifacts. If that is true, could you say that on the "Login to docker" line? Also, Can you make these steps into a numbered list?

Reply:
@jiayiwang7 It houses the EKS Anywhere packages artifacts. I'll update the text here to give that information